docs/getting-started/installing/troubleshooting.md

```diff
@@ -4,21 +4,18 @@ title: Troubleshooting
 
 # Services don't start
 
-## Symptons
+## Symptoms
 
 1. `kontena node ls` show nodes in a __OK__ status, but services are not starting.
-2. `kontena grid logs` show execeptions like the one below
+2. `kontena grid logs` show execeptions that looks like the one below
 
```

Comment:
Maybe add to "symptoms" something like:
- `kontena grid health` shows nodes with issues
- `kontena node health node-x` shows some issues